syntaxes/fsh.tmLanguage.json

```diff
@@ -30,7 +30,7 @@
 				},
 				{
 					"name": "keyword.reserved.fsh",
-					"match": "\\b(?<=\\s)(and|codes|contains|exclude|from|includes|is-a|is-not-a|named|obeys|only|or|system|units|valueset|where|D|MS|N|SU|TU|\\?!)(?=\\s)\\b"
+					"match": "\\b(?<=\\s)(and|codes|contains|exclude|from|include|is-a|is-not-a|descendent-of|regex|in|not-in|generalizes|exists|named|obeys|only|or|system|units|valueset|where|D|MS|N|SU|TU|\\?!)(?=\\s)\\b"
```

Comment:
Is there a particular order you added the VS filter keywords in? It looks like this list was previously in alphabetic order (minus the flags at the end). Are you just also trying to group these keywords together?

Reply:
No not really, this is just the order they appear in SUSHI. I will alphabetize.

Reply:
Ok @jafeltra see https://github.com/standardhealth/vscode-language-fsh/pull/9/commits/ce7e91bc1135c5db3608939573fd6820d7730a21 for the addition of `insert` and alphabetization.